src/TimeTrackerWindow.cpp

```diff
@@ -106,14 +106,33 @@ TimeTrackerWindow::MessageReceived(BMessage* message)
 	case BUTTON_NEW_TASK_OK:
 	{
 		BString	TaskName;
-		TaskName.SetTo(message->FindString("Title"));	
-		m_CardLayout->SetVisibleItem((int32)0);
-		m_ListView->AddItem(new TaskListItem(TaskName)); //Hey a new task. DOH!
-		InvalidateLayout();
-		ResizeTo(m_Frame.Width(), m_Frame.Height());
-		resume_thread(m_Thread);	//Okay back to work ;)
-		m_TaskMenu->SetEnabled(true);
-		break;
+		TaskName.SetTo(message->FindString("Title"));
+
+		int32 button_index;
```

Comment:
Worked while checking it multiple times probably because the garbage value was 0.